how-tos/releases-support/README.md

```diff
@@ -0,0 +1,86 @@
+---
+description: In order to give more visibility to our developers community, about agent usability and support in the future, here is the spot for important lifecycle dates per agent stack and versions.
+---
+
+# Releases Support
+
+## Introduction
+
+Each project using Forest Admin needs a minimal maintenance of the agent dependency installed since day 1.
+Such maintenance will ensure the best possible experience with the platform: new features, performance improvements, reactive support,...
+
+In order to give more visibility to our developers community, about agent usability and support in the future, we will share, in this section, the important lifecycle dates per agent stack and versions.
+
+## Definitions
+
+Before jumping in your favorite stack tab, you’ll find below the detailed definition of the agent lifecycle dates:
+
+| Date            | Definition                                                                                                                                                                                                                                                                                                                 |
+| --------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Initial Release | It marks the date of the official release of a new agent major version, and will, as a consequence, engage Forest Admin on a minimal 18 months period guaranty to keep the agent features 100% compatible with the platform, to provide maintenance, troubleshooting or support, to fix defects, bugs and vulnerabilities. |
+| End-of-Support  | At this date, Forest Admin will will no longer provide maintenance, troubleshooting or support, and will no longer fix defects, bugs or vulnerabilities.                                                                                                                                                                   |
+| End-of-Life     | At this date, Forest Admin will not guaranty the agent be usable. It is very likely that your agent integration will break your admin panel, as we will start removing some legacy features support on our platform.                                                                                                       |
+
+## Releases information per stack
+
+{% tabs %}
+{% tab title="Express Sequelize Agent" %}
```

Comment:
You say "Agent" here and for Express Mongoose, but not for Ruby on Rails, Django, etc.